batch/scheduler/KGenericScheduler.class.php

```diff
@@ -325,7 +333,11 @@ private function shouldExecute(KSchedularTaskConfig $taskConfig)
 				return false;
 		}
 
-		KalturaLog::debug("Worker [{$taskConfig->name}] id [{$taskConfig->id}] running batches [$runningBatches] max instances [{$taskConfig->maxInstances}]");
+		if ( !isset($this->lastWorkerLog[$taskConfig->id]) || (($this->logWorkerInterval >= 0) && (time() >= ($this->lastWorkerLog[$taskConfig->id] + $this->logWorkerInterval))) )
```

Comment:
If $this->logWorkerInterval == 0 it will log every time, make sure it's larger than 0.